test/validate-prowgen-breaking-changes.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+workdir="$( mktemp -d )"
+trap 'rm -rf "${workdir}"' EXIT
+
+git clone https://github.com/openshift/release.git --depth 1 "${workdir}/release"
+
+ci-operator-prowgen --from-dir "${workdir}/release/ci-operator/config" --to-dir "${workdir}/release/ci-operator/jobs"
+
+pushd "${workdir}/release"
+
+if [ -n "$(git status --porcelain)" ]; then
```

Comment:
Could also make this a `diff` a la the prowgen check script in release so the test image doesn't need `git`

Reply:
Could, but then I would need to make a pre-prowgen copy. I think we have git in the test image - if so, I'd like to keep this simple. If not, I'll change it.

Reply:
I meant output to a different dir and diff the release master state but if we already have `git` then it's fine